tests/numerics/data_out_rotation_postprocessor_scalar_02.cc

```diff
@@ -0,0 +1,124 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+// tests DataPostprocessor: access the cell we are currently working on
```

Comment:
There is actually no _01 test. All of the tests I'm adding were originally cloned from `data_out_postprocessor_01.cc`, and the comparison is always between `data_out_*_postprocessor_scalar_02.cc` and `data_out_*_postprocessor_vector_02.cc`.

So the purpose is encoded into the file name. But I've added this to the tests as well.